CHANGELOG.md

```diff
@@ -3,6 +3,23 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [[3.5.0](URL)] - Date
+
+This release expands the Illumina workflow by adding BLAST analysis, coverage plots, variant calling, and MultiQC reports. Modifications were made to existing modules, and new modules were added.
+
+### Changes
+
+* **feat**: Added variant calling, BLAST analysis, coverage plots, and MultiQC to the Illumina workflow to match the capabilities of the Nanopore workflow.
+* **feat**: Introduced a new module, Freebayes, for Illumina variant calling.
+* **refactor**: Rearranged the Illumina workflow to integrate the new changes and enhance compatibility.
+* **update**: Updated Minimap2 and BCFtools modules to better accommodate Illumina reads.
+* **config**: Changed process labels for IRMA and MultiQC modules to "long" to avoid timeouts for large short-read datasets.
```

Comment:
How large are these datasets to trigger timeouts? Do you see runtime exponentially increase with input reads file sizes?

Would a downsampling step be useful in these cases?

I think we tend to have file sizes in the 10-20MB range for `fastq.gz` files from Illumina. Our Nanopore reads are about the same size. Is there a benefit for much more sequence data than that?

Reply:
This issue was addressed in another comment. We will be investigating further in (https://github.com/CFIA-NCFAD/nf-flu/issues/81 and https://github.com/CFIA-NCFAD/nf-flu/issues/80).